### PR TITLE
Several fixes, fully connected graph construction CGENN, MIParT guarding, and pytest removal

### PR DESCRIPTION
**Pull request by t0mnt**
tests/lloca not in repo, only in library repo
mipart rapidity clamp now matches to_ptrapphim
lorentznet has  `n_scalar` not `in_s_channels`
raw logits were previously plotted in range `[0,1]` so histograms only showed that sliver of logits rather than showing sigmoid probabilities, now fixed

### CODE REVIEW

**Comment by t0mnt** [comment]
from a bug fix search of my own graph transformer repo, essentially organizing every fix that applies upstream too (and not to the graph transformers) into a separate branch and then PR'ing it.

**Comment by t0mnt** [comment]
In `reproduce.md` "Note that LLoCa/L-GATr taggers can also be used without xformers, but that requires modifying the data embedding and attention mask construction."
With flex/flash backends is this still true?

File: .github/workflows/tests.yaml [hunk 1]
```diff
@@ -28,5 +28,9 @@ jobs:
 
     - name: Run unit tests
       run: |
-        pytest tests/lloca
+        # tests/lloca does not exist in this repo (the lloca library tests live in the
+        # lloca repo), so this step errored on its first line and tests/experiments was
+        # run by no CI job at all. Run the tagging invariance suite instead; test_tag_flops
+        # stays out of CI because its learned-frame cases need a CUDA-matched xformers build.
+        pytest tests/experiments/test_tag_invariance.py
```
Comment (spinjo) [inline .github/workflows/tests.yaml]: Removing this test because we did not build it as a proper test that should pass 100%; we just used it for rough tests

**Review by spinjo — APPROVED** [review]
Thanks! All good catches, thanks for pushing them.

I made commits to remove a workflow, and mainly to remove the LLM-style explanatory commits that I don't like in a codebase once it works^^

**Comment by t0mnt** [comment]
thanks! I too find them annoying but I needed to sort through what commits on `experiments.py` were and weren't applicable to the repo (like changes to the table handling aren't but to fixing LorentzNet are) quickly

**Review by t0mnt — COMMENTED** [review]
why remove test_tag_invariance?

**Comment by t0mnt** [comment]
No assertions maybe?

**Comment by t0mnt** [comment]
the world may never know
